alacritty_terminal/src/config/mod.rs

```diff
@@ -128,6 +128,8 @@ pub struct Cursor {
     pub style: CursorStyle,
```

Comment:
We can't just change things, we need to still support the old format and print deprecation warnings.